src/classes/Category.php

```diff
@@ -8,38 +8,21 @@ class Category
     private $categoryName;
     private $defaultImageFilePath;
     private $defaultImageAlt;
-    private $path = "/src/app/categories.php?id=";
 
-    /**
-     * Generates html for category list.
-     *
-     * @return string
-     */
-    public function getCategoryListLink():string {
-        return "<a href='" .
-            $this->path .
-            $this->id .
-            "' ><h5>" .
-            $this->categoryName .
-            "</h5></a><br>";
+    public function getId() {
+        return $this->id;
     }
 
-    /**
-     * Generates html for category blocks.
-     *
-     * @return string
-     */
-    public function getCategoryImageLink():string {
-        return "<div class=\"category col-xs-10 col-sm-6 col-md-4\">
-                <a href='" .
-                $this->path .
-                $this->id .
-                "'><img class='img-thumbnail' src='" .
-                $this->defaultImageFilePath .
-                "' alt='" .
-                $this->defaultImageAlt .
-                "'><h4>" .
-                $this->categoryName .
-                "</h4></a></div>";
+    public function getCategoryName() {
+        return $this->categoryName;
     }
+
+    public function getDefaultImageFilePath() {
```

Comment:
{ on next line

Reply:
type hint return

Reply:
Have you tested what happens if a product has multiple images? Part of the story requirements is that a product can display multiple images. This will affect the return type hinting etc